src/js/components/molecules/social-sharing/social-sharing.ts

```diff
@@ -0,0 +1,44 @@
+/** @format */
+
+import { Vue, Component, Prop } from 'vue-property-decorator';
+// import { SocialSharingProps } from './social-sharing.models';
+import WithRender from './social-sharing.vue.html';
+import { generateSocialSharingUrl } from '../../../utils/generate-social-sharing-url';
+import BrowserCapabilities from '../../../utils/browser-capabilities';
+
+@WithRender
+@Component({
+    name: 'SocialSharing',
+})
+
+export class SocialSharing extends Vue {
+    // @Prop({ type: Array, required: true }) items!: SocialSharingProps['items'];
+
+    get items() {
+        return [
+            {
+                type: 'linkedin',
+                iconName: 'linkedin',
+                label: 'Share on Linkedin',
+            },
+            {
+                type: 'facebook',
+                iconName: 'facebook',
+                label: 'Share on Facebook',
+            },
+            {
+                type: 'twitter',
+                iconName: 'twitter',
+                label: 'Share on Twitter',
+            },
+        ];
+    }
+
+    url({ type }) {
```

Comment:
Not using the $route?

Reply:
Nope… did not seem to work… besides the `$route` only provided `fullPath` and would have to be prepended with the protocol and hostname.

Reply:
Ok then its fine